src/Microsoft.DotNet.XHarness.CLI/CommandArguments/XHarnessCommandArguments.cs

```diff
@@ -28,7 +28,9 @@ public OptionSet GetOptions()
         {
             var options = GetCommandOptions();
 
-            options.Add("verbosity=|v=", "Verbosity level (1-6) where higher means less logging. (default = 2 / Information)", v => Verbosity = ParseArgument<LogLevel>("verbosity", v));
+            options.Add("verbosity:|v:", "Verbosity level - defaults to 'Information'. If passed without value, 'Debug' is assumed (highest)",
```

Comment:
The wording around defaults is confusing to me, I assume it means that without -v it uses Info, but with -v it uses Debug (unless you pass an explicit value)?

How about this instead:
```suggestion
            options.Add("verbosity:|v:", "Verbosity level - defaults to 'Information' if not specified. If passed without value, 'Debug' is assumed (highest)",
```

Reply:
Yes, you're completely right. Thanks for the feedback!